walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -411,31 +413,75 @@ public void allOptionalAreIgnored() {
 
     Collection<TimeRange> actual = query.query(events, request);
     Collection<TimeRange> expected =
-        Arrays.asList(TimeRange.fromStartEnd(TIME_0830AM, TimeRange.END_OF_DAY, true));
+        Arrays.asList(TimeRange.fromStartEnd(TIME_0830AM, TIME_0900AM, false));
 
     Assert.assertEquals(expected, actual);
   }
 
   @Test
-  public void eventWithRequiredAndOptionalAttendees() {
-    // An event with both an optional and required attendee is present. Even though the
-    // optional attendee is not considered due to other conflicts, this event should still
-    // influence the time ranges.
+  public void optionalOptimizationForQuantity() {
+    // Three optional attendees are present. Although only considering one optional attendee
+    // would result in more available time ranges, two optional attendees can be considered in
+    // another optimization. The selection with the most optional attendees should be 
+    // preferred.
+
+    // Required:   |---|  |-|   |--|
+    // Optional 1:     |--|
+    // Optional 2:          |---|
+    // Optional 3:       |------|
+    //
+    // Correct:    |---| |---------| (Optional 2 and 3)
```

Comment:
Keep your notation consistent across all of your tests. If you're going to use `-` in the last line to denote open space reuse that convention in every single test. Otherwise the reader has to spend an enormous amount of time thinking about your comment to understand it.

Reply:
Good point. Even though I didn't write those above tests I should definitely be working to keep the conventions in place. Will look over my comments.